shared/message/behaviour/MotionCommand.proto

```diff
@@ -22,28 +22,26 @@ package message.behaviour;
 
 import "Vector.proto";
 
+/**
+ * Details of which walk behaviour to execute and where on the field to do it
+ */
 message MotionCommand {
-    // Defines the possible types of motion command:
     enum Type {
-        // Stop moving and just stand still, (equivalent to a VelocityCommand of {0} m/s).
+        /// Stop moving and just stand still (equivalent to a VelocityCommand of 0 m/s)
```

Comment:
```suggestion
        /// Stop moving and just stand still (equivalent to disabling the walk engine)
```
More accurate, might have been different in the past but QuinticWalk just disables it.